table/tables/state_remote.go

```diff
@@ -177,6 +181,9 @@ func (h *stateRemoteHandle) lockForWriteOnce(ctx context.Context, tid int64, ts
 				tid); err != nil {
 				return errors.Trace(err)
 			}
+			if err = h.commitTxn(ctx); err != nil {
```

Comment:
Why commit here?

Reply:
A misoperation when I merge master and resolve conflict.
Let me fix it.